README.md

```diff
@@ -199,25 +198,20 @@ class StackTest {
 markers. This way we can have java comments: `//`, python comments: `#` or
 similar preceding our markers. **This means code can still compile!**
 
-There are some rules for prefixing to observe:
+### There are some rules for prefixing to observe:
 
-`Sanitizer`:
+Prefixes are valid for a single block, and are defined on the same line as the `REPOBEE-SANITIZER-START` marker of that block. All text found before the `START` marker (stripped fom whitespace) counts as the prefix.
```

Comment:
```suggestion
Prefixes are valid for a single block, and are defined on the same line as the `REPOBEE-SANITIZER-START` marker of that block. The prefix is defined as all characters occurring before the `START` marker, without leading and trailing whitespace. Whitespace _within_ the prefix, such as in `/   /`, counts as part of the prefix.

Reply:
Note that I removed "stripped" here, because it's ambiguous. You even use it ambiguously within this PR. Here, it refers to removing leading and trailing whitespace, but when you talk about "stripping the prefix", it actually refers to just removing (has nothing to do with leading or trailing anything else).